server/app-engine/gtfsrt.py

```diff
@@ -420,6 +438,41 @@ def add_alert(datastore_client, alert):
     datastore_client.put(entity)
     print('+ wrote alert')
 
+def delete_alert(datastore_client, alert):
+    print('delete_alert()')
+    print('- alert: ' + str(alert))
+
+    query = datastore_client.query(kind='alert')
+    # We need to filter for all fields, since a unique ID isn't passed into the protobuf
+    query.add_filter('agency_key', '=', alert["agency_key"])
```

Comment:
Have you tested this? I'm pretty sure that this filter group would require an update to index.yaml, which I'm not seeing as part of this PR.

Reply:
It does work - I've never modified index.yaml and don't know a ton about it. I've tested for all of the field values, though, and it successfully deletes the correct alert.

Reply:
Ok, a bit more information about indexes, from [here](https://cloud.google.com/datastore/docs/concepts/indexes) (See "Index Configuration"):

**Built-in indexes are sufficient to perform many simple queries, such as equality-only queries and simple inequality queries.**